src/app/components/core/notifications/notification-service-modal/notification-service-modal.component.ts

```diff
@@ -0,0 +1,353 @@
+import { Component, EventEmitter, Output, OnChanges, Input, SimpleChanges, ViewChild, HostListener } from '@angular/core';
+import { FormBuilder, NgForm } from '@angular/forms';
+import { ProgressBarService, NotificationsService, AlertService, ServicesApiService, SchedulesService,
+  ConfigurationService } from '../../../../services';
+import {
+  ViewConfigItemComponent
+} from '../../configuration-manager/view-config-item/view-config-item.component';
+import { AlertDialogComponent } from '../../../common/alert-dialog/alert-dialog.component';
+import { isEmpty } from 'lodash';
+
+@Component({
+  selector: 'app-notification-service-modal',
+  templateUrl: './notification-service-modal.component.html',
+  styleUrls: ['./notification-service-modal.component.css']
+})
+export class NotificationServiceModalComponent implements OnChanges {
+  enabled: Boolean;
+  name: string;
```

Comment:
this name variable is not required we already have `notificationServiceName` variable that holds the service name. We can pass  `notificationServiceName` in html file in input field `[(ngModel)]='notificationServiceName'`.